src/Akka.Streams.Kafka/Messages/CommittableOffset.cs

```diff
@@ -10,22 +9,29 @@ namespace Akka.Streams.Kafka.Messages
     /// </summary>
     internal sealed class CommittableOffset : ICommittableOffsetMetadata
     {
-        /// <inheritdoc />
         public long BatchSize => 1;
+
+        public ICommittableOffsetBatch Updated(ICommittable offset)
+        {
+            // need to combine this offset AND offset to form a new batch
+            return CommittableOffsetBatch.Create([this, offset]);
+        }
+
         /// <summary>
         /// Offset value
         /// </summary>
         public GroupTopicPartitionOffset Offset { get; }
         /// <summary>
-        /// Cosumed record metadata
+        /// Consumed record metadata
         /// </summary>
         public string Metadata { get; }
+        
         /// <summary>
         /// Committer
         /// </summary>
-        public IInternalCommitter Committer { get; }
+        public KafkaAsyncConsumerCommitter Committer { get; }
```

Comment:
`IInternalCommitter` wasn't a super useful abstraction and I removed it - making the parts of the `KafkaAsyncConsumerCommitter` `virtual` where needed instead.